acts

```diff
@@ -146,10 +153,10 @@ backuprc=0 # Notice any failed backups
 for dir in $backuptargets; do
     log_debug "message=\"Starting backup of $dir\""
     archive_starttime=$(date +%s)
-    nicedirname=$(echo "$dir" | tr -d '/')
+    nicedirname=${dir:+-${dir#/}}  # /foo -> -foo, / -> ''
```

Comment:
`$dir` could have multiple `/` in it, for example a backuptarget of `home/aj`, and the new version doesn't seem to handle that:

```
$ docker run -ti alpine sh
/ # dir="home/aj" ; echo ${dir:+-${dir#/}}
-home/aj
```

Reply:
Ah! Yes, I misread it. I figured only the first / needed to be deleted.